src/server.js

```diff
@@ -10,7 +11,7 @@ const logger = require('./utils/logger');
 
 
 // Config
-const SERVER_PORT = process.env.PORT || config.get('SERVER.PORT');
+const SERVER_PORT = process.env.PORT;
```

Comment:
May use "const { SERVER_PORT } = process.env;" instead for streamlining with other services.

Reply:
Will change SERVER_PORT to PORT for streamlining.